boards/posix/native_posix/cmdline.c

```diff
@@ -36,6 +38,19 @@ static void cmd_seed_found(char *argv, int offset)
 }
 #endif
 
+#if defined(CONFIG_BT_USERCHAN)
+int bt_dev_index = -1;
+
+static void cmd_bt_dev_found(char *argv, int offset)
+{
+	if (strncmp(&argv[offset], "hci", 3) || strlen(&argv[offset]) < 4) {
+		return;
```

Comment:
Would it make sense to be a bit kinder to users and print a warning? (if things do not break after, or an error if they do). Something like:
posix_print_warning("Warning: the bt-dev command line option only accepts hci devices, for ex. --bt-dev:hci0 (you provided %s)\n", argv);

Reply:
That's a good suggestion. I'll add it.